### ISSUE TITLE
Unable to build Galera image - issue with qpress archive

### ISSUE THREAD

**Issue by BenResTech** [colinmollenhour/mariadb-galera-swarm#66]
Hi,

When trying to build from your dockerfile I get an error when attempting to extract the qpress archive. This fails the build step and so fails building the image.
I have manually checked and the archive is still downloadable from the quicklz website.

The full error message seen in the build is:
```
Processing triggers for libc-bin (2.27-3ubuntu1) ...
+ curl -sSL -o /tmp/qpress.tar http://www.quicklz.com/qpress-11-linux-x64.tar
+ tar -C /usr/local/bin -xf /tmp/qpress.tar qpress
tar: This does not look like a tar archive
tar: qpress: Not found in archive
tar: Exiting with failure status due to previous errors
```

Just an FYI in case you ever pop your build cache, as unless something changes you won't be able to build a new image.

Ben

UPDATE: So the reason that the tar command fails is that it has not curled down the tar, it has curled down the following error message:

`<h1>Security incident detected</h1><p>Your request was blocked. Please try again later (or don't).</p>`

Investigating further, ignore this issue for now as it may not be a problem for other users.

UPDATE2: I cannot curl the archive down onto an AWS instance either. This confirms that it is not a security issue relating to my local machine or network. Can anyone verify that they are able to curl the qpress archive in the same way that the Dockerfile build step does?

**Comment by BenResTech** [comment]
As a workaround for now I've just downloaded the qpress archive locally and then added a copy step to load it into the docker build environment. Not as neat as downloading it fresh each time, but at least now it builds.

**Comment by BenResTech** [comment]
Found the problem, it appears that www.quicklz.com no longer allows downloads from curl or wget's User Agents (or from blank ones). Presumably they were having problems with site scraping or something.

Either way, this can be fixed by changing the curl command to declare the user agent as being from a browser.

Creating a PR to fix this - #67

**Comment by BenResTech** [comment]
Closing as binary is included in repo now.
